docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -0,0 +1,259 @@
+  name text,
+  department text,
+  change_date date
+);
+```
+
+*change_date* represents the date when any type of change occured on the employee record. This date is typically required when generating monthly reports and only the last month's data is queried. To reduce the amount of old data that  needs to be stored, You can reduce the amount of data to store and keep these data for one year, and on the first of every month you remove the data from the previous month. You can partition the `employees` table to meet these requirements. You start by specifying bounds corresponding to the partitioning method and partition key of the `employees` table. This means you create partitions as regular tables and YSQL generates partition constraints automatically based on the partition bound specification every time they have to be referenced.
+
+You can declare partioning on the `employees` table by first creating it as a partitioned table. You specify the `PARTITION BY` clause and supply it with the partitioning method such as `RANGE`  as well as a list of columns as a partition key, as shown in the following example:
+
+```sql
+CREATE TABLE employees (
+  employee_no integer PRIMARY KEY,
+  name text,
+  department text,
+  change_date date
+) 
+PARTITION BY RANGE (change_date);
+```
+
+You create the actual partitions as follows:
+
+```sql
+CREATE TABLE employees_2019_02 PARTITION OF employees
+  FOR VALUES FROM ('2019-02-01') TO ('2019-03-01');
+
+CREATE TABLE employees_2019_03 PARTITION OF employees
+  FOR VALUES FROM ('2019-03-01') TO ('2019-04-01');
+
+...
+CREATE TABLE employees_2020_11 PARTITION OF employees
+  FOR VALUES FROM ('2020-11-01') TO ('2020-12-01');
+
+CREATE TABLE employees_2020_12 PARTITION OF employees
+  FOR VALUES FROM ('2020-12-01') TO ('2021-01-01')
+  TABLESPACE fasttablespace;
+
+CREATE TABLE employees_2021_01 PARTITION OF employees
+  FOR VALUES FROM ('2021-01-01') TO ('2021-02-01')
+  TABLESPACE fasttablespace;
+```
+
+Optionally, you can create an index on the key columns and other indexes for every partition, as follows:
+
+```sql
+CREATE INDEX ON employees_2019_02 (change_date);
+CREATE INDEX ON employees_2019_03 (change_date);
+...
+CREATE INDEX ON employees_2020_11 (change_date);
+CREATE INDEX ON employees_2020_12 (change_date);
+CREATE INDEX ON employees_2021_01 (change_date);
+```
+
+Partitioning is a flexible technique that allows you to remove old partitions and add new partitions for  new data when required. You do this by changing the partition structure instead of the actual data.
+
+The following example shows how to quickly remove old data via dropping the partition (you need to take an `ACCESS EXCLUSIVE` lock on the partitioned `employees` table):
+
+```sql
+DROP TABLE employees_2019_02;
+```
+
+The following example shows how remove the partition from the partitioned table while retaining access to it as a separate table which enables you to performed operatioins on the data:
```

Comment:
"how remove" --> "how to remove"
"operatioins" -- > operations

Reply:
fixed